### PR TITLE
[Task06]

### PR DESCRIPTION
**Pull request by ando-andy**
## Related Tickets

- [#Ticket ID](https://docs.google.com/spreadsheets/d/1F5V2sIUMGej3Wrp8qYRscf0KaVSlccBmqkRx49_hUVo/edit#gid=935273790)

## What's this PR do ?
Create MovieDetail

## Library
*(List gem, library third party add new)*
https://github.com/ando-andy/MGCleanArchitecture
https://github.com/evgenyneu/Cosmos

## Impacted Areas in Application
*(List features, api, models or services that this PR will affect)*

## Checklist

- [x] It was tested in local success ?
- [x] Updated rake task, environment variable at [here](https://???)
- [x] Fill link PR into ticket and the opposite
- [x] Note reason, scope of influence, solution into ticket
- [x] Validate UI/Model/API

## Notes
*(Other notes)*

### CODE REVIEW

File: MovieDB/Scenes/Movie/MoviesViewModel.swift [hunk 1]
```diff
@@ -95,6 +99,24 @@ extension MoviesViewModel: ViewModelType {
             })
             .mapToVoid()
         
+        let selectedBanner = input.selectedBannerTrigger
+            .withLatestFrom(movieBannerList) {
+                return ($0, $1)
```
Comment (longvt-1468) [inline MovieDB/Scenes/Movie/MoviesViewModel.swift]:
```suggestion
                return $1[$0.row]
```
remove 
` .map { (indexPath, movieBanners) in
                return movieBanners[indexPath.row]
            }
`

File: MovieDB/Scenes/MovieDetail/MovieDetailViewController.swift [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    
+    func collectionView(_ collectionView: UICollectionView,
+                        layout collectionViewLayout: UICollectionViewLayout,
+                        sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let height = collectionView.frame.size.height
+        let width = height / 2
+        return CGSize(width: width, height: height)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
```
Comment (longvt-1468) [inline MovieDB/Scenes/MovieDetail/MovieDetailViewController.swift]: break line

File: MovieDB/Scenes/Movie/MoviesViewModel.swift (hunk 1 above)
Comment (ando-andy) [reply MovieDB/Scenes/Movie/MoviesViewModel.swift]: Modified & removed🙇‍♂️

File: MovieDB/Scenes/MovieDetail/MovieDetailViewController.swift (hunk 2 above)
Comment (ando-andy) [reply MovieDB/Scenes/MovieDetail/MovieDetailViewController.swift]: Line-Break done🙇‍♂️

**Pull request merged**